docs/hub/spaces-more-ways-to-create.md

```diff
@@ -6,4 +6,8 @@ The [Repo Duplicator](https://huggingface.co/spaces/osanseviero/repo_duplicator)
 
 ## Creating a Space from a model
 
-The [Model Comparator Space Builder](https://huggingface.co/spaces/farukozderim/Model-Comparator-Space-Builder) can be used to create a Space directly from any model hosted on the Hub.
+New! You can now create a Gradio demo directly from most model pages, using the "Deploy -> Spaces" button.
+
+<video src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/model-page-deploy-to-spaces.mp4" controls autoplay muted loop />
```

Comment:
arf, this doesn't seem to work: 
<img width="1093" alt="image" src="https://user-images.githubusercontent.com/326577/207643207-ece5b097-38d5-4f4f-a2ff-018e72248e4f.png">

@mishig25 when you're back let me know if you think of a way to enable this